shared/models/server/custom-config.model.ts

```diff
@@ -57,6 +57,9 @@ export interface CustomConfig {
       miniature: {
         preferAuthorDisplayName: boolean
       }
+      resumableUpload: {
```

Comment:
If you don't plan to expose this setting in admin config, you don't need to add it to `custom-config.model.ts`